alembic/versions/20750539a335_add_ticket_to_families.py

```diff
@@ -23,7 +23,7 @@
 Base = declarative_base()
 
 
-class Family(Base):
+class Case(Base):
     __tablename__ = "family"
```

Comment:
```suggestion
    __tablename__ = "case"
```
are we going to change the table name in the database as well?

Reply:
Yes! But I'll leave it for a separate PR - might be possible to group the changes in a single revision